tenets/codelingo/go/deprecated/codelingo.yaml

```diff
@@ -0,0 +1,18 @@
+tenets:
+  - name: deprecated
+    flows:
```

Comment:
s/flows/actions

Reply:
Does lingo init still generate with "flows"?

Reply:
Ooh, yeah it does. And it seems to work too 🤔

Reply:
We support both, so that's good!